packages/itinerary-body/src/util.ts

```diff
@@ -102,7 +102,10 @@ export function getPlaceName(
 ): string {
   // If address is provided (i.e. for carshare station, use it)
   if (place.address) return place.address.split(",")[0];
-  if (place.networks && place.vertexType === "VEHICLERENTAL") {
+  if (
+    place.networks &&
+    (place.vertexType === "VEHICLERENTAL" || place.vertexType === "BIKESHARE")
```

Comment:
Is this the entirety of the patch beyond moving that method into transitive-overlay?